website_product_share/README.rst

```diff
@@ -0,0 +1,57 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+    :alt: License
+
+Product Share
+=============================
```

Comment:
This line should have the same length as the title. And please use latest README template from https://raw.githubusercontent.com/OCA/maintainer-tools/master/template/module/README.rst (that includes an upper line for example).